e2e/tests/noticeOfChange_test.js

```diff
@@ -1,29 +1,51 @@
 
 let caseId;
+let privateSolicitorDetails;
 
 Feature('Notice of change');
 
-BeforeSuite(async ({I}) => {
-  caseId = await I.submitNewCaseWithData(mandatorySubmissionFields);
+BeforeSuite(async () => {
+  privateSolicitorDetails = await apiHelper.getUser(config.privateSolicitorOne);
 });
 
-Scenario('Private solicitor obtains case access through NoC', async ({I, caseListPage, noticeOfChangePage}) => {
+Scenario('Private solicitor obtains access to an unrepresented case', async ({I, caseListPage, caseViewPage, noticeOfChangePage}) => {
+  caseId = await I.submitNewCaseWithData(mandatorySubmissionFields);
   await I.signIn(config.privateSolicitorOne);
   I.navigateToCaseList();
   caseListPage.searchForCasesWithId(caseId);
   I.dontSeeCaseInSearchResult(caseId);
-  noticeOfChangePage.navigate();
-  await noticeOfChangePage.enterCaseReference(caseId);
-  await I.retryUntilExists(() => I.click('Continue'), noticeOfChangePage.fields.applicantName);
-  await noticeOfChangePage.enterApplicantName('Swansea City Council');
-  noticeOfChangePage.enterRespondentName('Joe', 'Bloggs');
-  await I.retryUntilExists(() => I.click('Continue'), noticeOfChangePage.fields.confirmNoC);
-  noticeOfChangePage.confirmNoticeOfChange();
-  await I.retryUntilExists(() => I.click('Submit'), '.govuk-panel--confirmation');
-  I.see('Notice of change successful');
+
+  await noticeOfChangePage.userCompletesNoC(caseId, 'Swansea City Council', 'Joe', 'Bloggs');
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  assertRepresentative(I, privateSolicitorDetails.forename, privateSolicitorDetails.surname, privateSolicitorDetails.email, 'Private solicitors');
+});
+
+Scenario('Private solicitor replaces respondent solicitor on a represented case', async ({I, caseListPage, caseViewPage, noticeOfChangePage}) => {
+  caseId = await I.submitNewCaseWithData(representedCase);
+  await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId);
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  assertRepresentative(I, 'Tom', 'Jones', 'test@test.co.uk');
+
+  await I.signIn(config.privateSolicitorOne);
   I.navigateToCaseList();
   caseListPage.searchForCasesWithId(caseId);
-  I.seeCaseInSearchResult(caseId);
+  I.dontSeeCaseInSearchResult(caseId);
+
+  await noticeOfChangePage.userCompletesNoC(caseId, 'Swansea City Council', 'Joe', 'Bloggs');
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  assertRepresentative(I, privateSolicitorDetails.forename, privateSolicitorDetails.surname, privateSolicitorDetails.email, 'Private solicitors');
 });
+
+const assertRepresentative = (I, firstName, lastName, email, organisation) => {
+  I.seeInTab(['Representative', 'Representative\'s first name'], firstName);
+  I.seeInTab(['Representative', 'Representative\'s last name'], lastName);
+  I.seeInTab(['Representative', 'Email address'], email);
+
+  if (organisation) {
+    I.seeOrganisationInTab(['Respondents 1', 'Representative', 'Name'], organisation);
```

Comment:
you might just need a wait here, as the AAT/preview env is running crazy slow to load the organisation details, like
```suggestion
    I.waitForText(organisation);
    I.seeOrganisationInTab(['Respondents 1', 'Representative', 'Name'], organisation);
```

Reply:
Added